catalyst_voices/lib/pages/workspace/rich_text/problem_statement.dart

```diff
@@ -0,0 +1,6 @@
+const problemStatement = [
```

Comment:
consider to move all these samples in a subfolder of `rich_text` folder

Reply:
What do you mean? They are currently in subdirectory `rich_text`

Reply:
The idea is to separate "samples" from "code", i.e. in rich_text you can place the widgets and their utils, in `rich_text/samples` you can add the json samples.

Reply:
tbh, I can't see value in doing that. Especially that's only temporary solution.